src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputEmptyLineSeparatorFormerException.java

```diff
@@ -1,6 +0,0 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
-
-public class InputEmptyLineSeparatorFormerException
-{
-
-}
```

Comment:
please keep git history.
you need to do all by git commands "git mv oldpath newpath"  after that do  "git oldname newname" (2 commands).